components/Votes/ActiveVotes.tsx

```diff
@@ -0,0 +1,396 @@
+import { useConnectWallet } from "@web3-onboard/react";
+import {
+  Button,
+  IconWrapper,
+  Pagination,
+  Tooltip,
+  VotesList,
+  VotesListItem,
+  VotesTableHeadings,
+  VoteTimeline,
+} from "components";
+import { defaultResultsPerPage } from "constant";
+import { formatVotesToCommit } from "helpers";
+import { config } from "helpers/config";
+import {
+  useCommitVotes,
+  useContractsContext,
+  useDelegationContext,
+  usePanelContext,
+  useRevealVotes,
+  useStakingContext,
+  useUserContext,
+  useVotesContext,
+  useVoteTimingContext,
+  useWalletContext,
+} from "hooks";
+import { useEffect, useState } from "react";
+import { SelectedVotesByKeyT, VoteT } from "types";
+import {
+  ButtonInnerWrapper,
+  ButtonOuterWrapper,
+  ButtonSpacer,
+  Divider,
+  InfoText,
+  PaginationWrapper,
+  RecommittingVotesMessage,
+  Title,
+  VotesTableWrapper,
+  WarningIcon,
+} from "./style";
+
+export function ActiveVotes() {
+  const { activeVotesList, getUserDependentIsFetching } = useVotesContext();
+  const { phase, roundId } = useVoteTimingContext();
+  const { address, hasSigningKey, correctChainConnected, signingKey } =
+    useUserContext();
+  const { signer, sign, isSigning, setCorrectChain, isSettingChain } =
+    useWalletContext();
+  const { votingWriter } = useContractsContext();
+  const { stakedBalance } = useStakingContext();
+  const { getDelegationStatus, getDelegatorAddress } = useDelegationContext();
+  const { openPanel } = usePanelContext();
+  const [{ connecting: isConnectingWallet }, connect] = useConnectWallet();
+  const { commitVotesMutation, isCommittingVotes } = useCommitVotes();
+  const { revealVotesMutation, isRevealingVotes } = useRevealVotes();
+  const [selectedVotes, setSelectedVotes] = useState<SelectedVotesByKeyT>({});
+  const [dirtyInputs, setDirtyInput] = useState<boolean[]>([]);
+  const [votesToShow, setVotesToShow] = useState(activeVotesList);
```

Comment:
the `activeVotesList` is now the only list of votes that this component ever deals with